src/sentry/receivers/onboarding.py

```diff
@@ -0,0 +1,224 @@
+                task=OnboardingTask.INVITE_MEMBER,
+                user=user,
+                status=OnboardingTaskStatus.PENDING,
+                date_completed=timezone.now(),
+                data={ 'invited_member_id': member.id }
+            )
+    except IntegrityError:
+        pass
+
+@member_joined.connect(weak=False)
+def record_member_joined(member, **kwargs):
+    try:
+        OrganizationOnboardingTask.objects.filter(
+            organization=member.organization,
+            task=OnboardingTask.INVITE_MEMBER
+        ).exclude(
+            status=OnboardingTaskStatus.COMPLETE
+        ).update(
+            status=OnboardingTaskStatus.COMPLETE,
+            date_completed=timezone.now(),
+            data={'invited_member_id': member.id}
+        )
+    except IntegrityError:
+        pass
+
+@event_processed.connect(weak=False)
+def record_release_received(project, group, event, **kwargs):
+    if event.get_tag('sentry:release'):
+        try:
+            with transaction.atomic():
+                OrganizationOnboardingTask.objects.create(
+                    organization=project.organization,
+                    task=OnboardingTask.RELEASE_TRACKING,
+                    status=OnboardingTaskStatus.COMPLETE,
+                    project=project,
+                    date_completed=timezone.now()
+                )
+        except IntegrityError:
+            pass
+
+@event_processed.connect(weak=False)
+def record_user_context_received(project, group, event, **kwargs):
+    if event.data.get('sentry.interfaces.User'):
+        try:
+            with transaction.atomic():
+                OrganizationOnboardingTask.objects.create(
+                    organization=project.organization,
+                    task=OnboardingTask.USER_CONTEXT,
+                    status=OnboardingTaskStatus.COMPLETE,
+                    project=project,
+                    date_completed=timezone.now()
+                )
+        except IntegrityError:
+            pass
+
+@event_processed.connect(weak=False)
+def record_sourcemaps_received(project, group, event, **kwargs):
+    if has_sourcemap(event):
+        try:
+            with transaction.atomic():
+                OrganizationOnboardingTask.objects.create(
+                    organization=project.organization,
+                    task=OnboardingTask.SOURCEMAPS,
+                    status=OnboardingTaskStatus.COMPLETE,
+                    project=project,
+                    date_completed=timezone.now()
+                )
+        except IntegrityError:
+            pass
+
+@plugin_enabled.connect(weak=False)
+def record_plugin_enabled(plugin, project, user, **kwargs):
+    if isinstance(plugin, IssueTrackingPlugin):
+        task = OnboardingTask.ISSUE_TRACKER
+        status = OnboardingTaskStatus.PENDING
+    elif isinstance(plugin,  NotificationPlugin):
+        task = OnboardingTask.NOTIFICATION_SERVICE
+        status = OnboardingTaskStatus.COMPLETE
```

Comment:
There's an off chance that there is an `else` that gets hit here, if `plugin` is neither `IssueTrackingPlugin` or `NotificationPlugin`, which would end up hitting a `NameError` below. You should do a final:

```python
else:
  return
```